app/frontend/src/utils/timeAgo.ts

```diff
@@ -1,15 +1,15 @@
-const minuteMillis = 60000;
-const quaterHourMillis = 900000;
-const hourMillis = 3600000;
-const twoHourMillis = 7200000;
-const dayMillis = 86400000;
-const twoDayMillis = 172800000;
-const weekMillis = 604800000;
-const twoWeekMillis = 1209600000;
-const monthMillis = weekMillis * 4;
-const twoMonthMillis = weekMillis * 8;
-const yearMillis = 31557600000;
-const twoYearMillis = 31557600000 * 2;
+export const minuteMillis = 60000;
+export const quaterHourMillis = 900000;
```

Comment:
```suggestion
export const quarterHourMillis = 900000;
```
Nitpick: spelling mistake